docs/api/tensorflow_usage.rst

```diff
@@ -35,18 +48,16 @@ Limiting GPU Memory Growth
 ^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 TensorFlow by default blocks all the available GPU memory for the running process. This can be limiting if you are running
-multiple TensorFlow processes and want to distribute memory across them. To prevent this,
-set the environment variable ``TF_FORCE_GPU_ALLOW_GROWTH`` to ``True``.
+multiple TensorFlow processes and want to distribute memory across them. To prevent Rasa OS from blocking the
+complete available GPU memory, set the environment variable ``TF_FORCE_GPU_ALLOW_GROWTH`` to ``True``.
```

Comment:
```suggestion
multiple TensorFlow processes and want to distribute memory across them. To prevent Rasa Open Source from blocking all
of the available GPU memory, set the environment variable ``TF_FORCE_GPU_ALLOW_GROWTH`` to ``True``.
```